api4/cloud.go

```diff
@@ -115,18 +112,6 @@ func changeSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write(json)
 }
 
-func getSubscriptionStats(c *Context, w http.ResponseWriter, r *http.Request) {
-	s, err := c.App.GetSubscriptionStats()
-	if err != nil {
-		c.Err = err
-		return
-	}
-
-	stats, _ := json.Marshal(s)
-
-	w.Write([]byte(string(stats)))
-}
```

Comment:
I'm wondering if the basic CRUD (or at least the Read,Update) endpoints for the subscription should remain, but simply strip out the parts that mention User Limits? We're almost certainly going to need a GetSubscription (for admins) and GetSubscriptionStats (for non-admins) in the 7.0 product

Reply:
>  I think we should also remove the SubscriptionStats struct from the model.

+1 I believe the original purpose of this was to be used in unauthenticated sessions so this was the sanitized subscription info. We will no longer do any subscription checks without a session so I agree that the subscription object would include all needed data points.